cmd/talosctl/cmd/talos/config.go

```diff
@@ -186,6 +188,106 @@ var configAddCmd = &cobra.Command{
 	},
 }
 
+// configRemoveCmdFlags represents the `config remove` command flags.
+var configRemoveCmdFlags struct {
+	noconfirm bool
+	dry       bool
+}
+
+// configRemoveCmd represents the `config remove` command.
+var configRemoveCmd = &cobra.Command{
+	Use:   "remove <context>",
+	Short: "Remove a context",
+	Args:  cobra.ExactArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		pattern := args[0]
+		if pattern == "" {
+			return fmt.Errorf("no context specified")
+		}
+
+		c, err := clientconfig.Open(GlobalArgs.Talosconfig)
+		if err != nil {
+			return fmt.Errorf("error reading config: %w", err)
+		}
+
+		if len(c.Contexts) == 0 {
+			return errors.New("no contexts defined")
+		}
+
+		matches := maps.Keys(
+			maps.Filter(c.Contexts, func(context string, _ *clientconfig.Context) bool { return glob.Glob(pattern, context) }),
+		)
+		if len(matches) == 0 {
+			return fmt.Errorf("no contexts matched %q", pattern)
+		}
+
+		sort.Slice(matches, func(i, j int) bool { return matches[i] < matches[j] })
+
+		if len(c.Contexts) == 1 && matches[0] == c.Context {
+			if !configRemoveCmdFlags.noconfirm {
+				promt := fmt.Sprintf("Are you sure you want to remove the only %q context?", c.Context)
+
+				if !helpers.Confirm(promt) || configRemoveCmdFlags.dry {
+					return nil
+				}
+			}
+
+			return removeConfig(c)
+		}
+
+		noChanges := true
+
+		for _, match := range matches {
+			if match == c.Context {
+				fmt.Fprintf(os.Stderr, "Skipping removal of the current context %q, please change it to another before removing\n", match)
+
+				continue
+			}
+
+			if !configRemoveCmdFlags.noconfirm {
+				prompt := "Remove context " + match
+				if match == c.Context {
+					prompt += "(*)"
+				}
+
+				if !helpers.Confirm(prompt + "?") {
+					continue
+				}
+			} else {
+				fmt.Fprintln(os.Stderr, "Removing context", match)
```

Comment:
same here and other places - 
```suggestion
				fmt.Fprintln(os.Stderr, "removing context", match)
```